config/crd/bases/kuadrant.io_dnsrecords.yaml

```diff
@@ -15,7 +15,12 @@ spec:
     singular: dnsrecord
   scope: Namespaced
   versions:
-  - name: v1
+  - additionalPrinterColumns:
+    - description: DNSRecord ready.
+      jsonPath: .status.conditions[?(@.type=="Ready")].status
+      name: Ready
+      type: string
+    name: v1
```

Comment:
should we bump this up to v2,
or bump it down to something like v1alpha1 to reflect it's subject to change?

Reply:
I don't think we should make it v2, but yeah, all these APIs should probably be v1alpha1 for now

Reply:
Chnaged to v1alpha1